app/javascript/widget/components/layouts/ViewWithHeader.vue

```diff
@@ -1,56 +1,57 @@
 <template>
   <div
-    class="w-full h-full flex flex-col relative bg-slate-50 dark:bg-slate-800"
+    class="w-full h-full bg-white dark:bg-slate-900"
```

Comment:
@nithindavid The message view and header background colors are the same. Before, the chat background color was `dark:bg-slate-800`.

<img width="447" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/9922a49f-e157-42ee-bc2b-eb121346c1bb">